addons/website/tests/test_grid_layout.py

```diff
@@ -29,3 +29,16 @@ def test_01_replace_grid_image(self):
             'datas': base64.b64encode(req.content),
         })
         self.start_tour(self.env['website'].get_client_action_url('/'), 'website_replace_grid_image', login="admin")
+
+    def test_02_scroll_to_new_grid_item(self):
+        IrAttachment = self.env['ir.attachment']
+        base = "http://%s:%s" % (HOST, config['http_port'])
+        req = self.opener.get(base + '/web/image/website.s_banner_default_image')
+        IrAttachment.create({
+            'public': True,
+            'name': 's_banner_default_image.jpg',
+            'type': 'binary',
+            'res_model': 'ir_ui_view',
+            'datas': base64.b64encode(req.content),
+        })
```

Comment:
It's not the first time we do this no ? Shouldn't we create an util to handle that ?